doc/build/changelog/migration_20.rst

```diff
@@ -1687,7 +1687,7 @@ returning the main entity object, and not the extra column, use the
     stmt = select(User, Address.email_address).join(User.addresses).\
         distinct().order_by(Address.email_address)
 
-    result = session.execute(stmt).scalars(User).all()
+    result = session.execute(stmt).columns(User).all()
```

Comment:
oh i used scalars() there.   OK it can be either one depending on what you want back but yes let's keep it consistent.